docs/DeveloperGuide.md

```diff
@@ -707,3 +707,214 @@ Coming soon
 ### Saving data
 
 Coming soon
+
+--------------------------------------------------------------------------------------------------------------------
+
+## **Appendix: Planned Enhancements**
+
+Given below are the planned enhancements for WedLog, beyond v1.4. This list is not ranked in order of importance.
+
+### 1. Allow special characters in names of guests and vendors
+#### Description
+Currently, WedLog only allows names of guests and vendors to be alphanumeric. However, this may not be ideal as some names may contain special characters such as hyphens, apostrophes, and
+spaces. For example, the name "Mary-Ann O'Neil" is currently not allowed. This enhancement will allow such names to be added to WedLog. 
+
+#### Benefits
+We understand that some names may contain special characters. 
+This enhancement will allow users to add names of guests and vendors that contain special characters. This will allow WedLog to be more flexible in terms of the names that can be added to it.
+
+#### Implementation
+The implementation of this enhancement will be done in the `Name` class. By changing the validation regex of a name to allow special characters, users can input special characters in the names
+of guests and vendors.
+
+<br>
+
+### 2. Provide more specific warning and error messages for invalid commands
+#### Description
```

Comment:
do we want to differentiate between warnings and error messages? so warnings for valid commands with possible typos/unintended inputs, and error messages for invalid commands

Reply:
Let's leave it as error messages.